src/clinical/service-worker-thread/runner.ts

```diff
@@ -0,0 +1,17 @@
+import { StaticPool } from 'node-worker-threads-pool';
+import { WorkerTasks } from './tasks';
+
+const poolSize = 1;
+const poolTimeOutMs = 20 * 1000;
+const jsWorkerLocation = __dirname + '/worker.js';
+
+const pool = new StaticPool({
+  size: poolSize,
+  task: jsWorkerLocation,
+});
+
+export async function runTaskInWorkerThread(taskToRun: WorkerTasks, taskArgs: any) {
```

Comment:
from what I understand this is designed to be generic right

Reply:
yes, that is the idea. I tried to keep it generic so when we add other data export functions they can be non blocking too.